aspnetcore/mvc/controllers/filters.md

```diff
@@ -277,147 +347,157 @@ The [short circuiting resource filter](#short-circuiting-resource-filter) shown
 
 *Action filters*:
 
-* Implement either the `IActionFilter` or `IAsyncActionFilter` interface.
+* Implement either the <xref:Microsoft.AspNetCore.Mvc.Filters.IActionFilter> or <xref:Microsoft.AspNetCore.Mvc.Filters.IAsyncActionFilter> interface.
 * Their execution surrounds the execution of action methods.
 
-Here's a sample action filter:
+The following code shows a sample action filter:
 
-[!code-csharp[](./filters/sample/src/FiltersSample/Filters/SampleActionFilter.cs?name=snippet_ActionFilter)]
+[!code-csharp[](./filters/sample/FiltersSample/Filters/SampleActionFilter.cs?name=snippet_ActionFilter)]
 
 The <xref:Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext> provides the following properties:
 
-* `ActionArguments` - lets you manipulate the inputs to the action.
-* `Controller` - lets you manipulate the controller instance. 
-* `Result` - setting this short-circuits execution of the action method and subsequent action filters. Throwing an exception also prevents execution of the action method and subsequent filters, but is treated as a failure instead of a successful result.
+* <xref:Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext.ActionArguments> - enables the inputs to an action method be read or modified.
+* <xref:Microsoft.AspNetCore.Mvc.Controller> - enables manipulating the controller instance.
+* <xref:System.Web.Mvc.ActionExecutingContext.Result> - setting `Result` short-circuits execution of the action method and subsequent action filters.
+
+Throwing an exception in an action method:
+
+* Prevents execution of the action method and subsequent filters.
+* Unlike setting `Result`, is treated as a failure instead of a successful result.
 
 The <xref:Microsoft.AspNetCore.Mvc.Filters.ActionExecutedContext> provides `Controller` and `Result` plus the following properties:
 
-* `Canceled` - will be true if the action execution was short-circuited by another filter.
-* `Exception` - will be non-null if the action or a subsequent action filter threw an exception. Setting this property to null effectively 'handles' an exception, and `Result` will be executed as if it were returned from the action method normally.
+* <xref:System.Web.Mvc.ActionExecutedContext.Canceled> - True if the action execution was short-circuited by another filter.
```

Comment:
OCD alert: For the `ActionExecutingContext` section, the first letters after are lowercase.